### ISSUE TITLE
CSS style for list items - bullet or number appears on last line instead of first line of list item

### ISSUE THREAD

**Issue by sarahmaddox** [kubeflow/website#163]
In a list, the bullet or number often appears on last line of the list item instead of on the first line of the list item

This happens on the blog as well as in the docs.

Blog post: https://www.kubeflow.org/blog/announcing_kubeflow_0.2/
![screen shot 2018-08-16 at 8 47 46 am](https://user-images.githubusercontent.com/6138251/44177494-90d2be80-a131-11e8-8ce2-f6f39f1981bc.png)

Doc: https://www.kubeflow.org/docs/started/getting-started-gke/
![screen shot 2018-08-16 at 8 49 41 am](https://user-images.githubusercontent.com/6138251/44177523-a811ac00-a131-11e8-92cf-6b25f6c3935d.png)

The problem seems to depend on spacing in the list: Sometimes you can make the bullet/number appear in the right place by removing all empty lines between list items. But that's not possible if you need code snippets in the list, because the lack of spacing makes the code snippets render as normal text.

**Comment by sarahmaddox** [comment]
Another list-related style that needs improving: The styling doesn't handle 3 levels of indent, for example, the markup below renders as a numbered list item, then a bullet list item, then a list item marked by a large rectangular icon:

```
1. Choose one of the options below to connect to JupyterHub:

    * To connect to JupyterHub using the Kubeflow URL:

        1. Open a web browser and go to your Kubeflow URL.
```

**Comment by sarahmaddox** [comment]
I've let the designer know about these issues.

**Comment by sarahmaddox** [comment]
The designer has given us a set of updated CSS styles, which largely fix the first problem - list icons now appear on the first line of a multi-line list in most cases. The fix also fixes the second problem reported on this issue: styling for three levels of indent.

@abhi-g has submitted the designer's fixes in this PR:
https://github.com/kubeflow/website/pull/192

**Comment by sarahmaddox** [comment]
However, the first problem (list icons appearing on last line of multi-line list) still occurs under certain circumstances.

See the section "Improved getting started experience" on this page:
https://deploy-preview-192--competent-brattain-de2d6d.netlify.com/blog/announcing_kubeflow_0.2/

Also under "Using your own domain" here:
https://deploy-preview-192--competent-brattain-de2d6d.netlify.com/docs/started/getting-started-gke/

The problem seems to happen now only when code blocks are interspersed between the list items.

I'll get in touch with the designer again.

**Comment by sarahmaddox** [comment]
I'm closing this issue because we've fixed most of the problem, and issue #355 covers the remaining issue with bullet items.

/close
